charts/redpanda/templates/post-install-upgrade-job.yaml

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ template "redpanda.fullname" . }}-post-install
+  name: {{ template "redpanda.fullname" . }}-configurator
```

Comment:
Does this do more bootstrap configuration? How about `bootstrap-configurator`?

Reply:
It re-configures things, too, which is why it's no longer just post-install.

If we're going to be pedantic, let's use real words: configuration, cluster-configuration, something like that?

Reply:
Alright i think configuration makes more sense. Do we expect SASL superusers list to be ever changed/updated/appended in the future? Just making a mental note.

Reply:
> Do we expect SASL superusers list to be ever changed/updated/appended in the future?

I think we should allow that. User should be able to change list of superusers including:
* changing the password
* removing or replacing superusers